core/src/skybox.cpp

```diff
@@ -0,0 +1,166 @@
+#include "knoting/skybox.h"
+
+namespace knot {
+namespace components {
+
+struct PosColorTexCoord0Vertex {
+    float m_x;
+    float m_y;
+    float m_z;
+    uint32_t m_rgba;
+    float m_u;
+    float m_v;
+
+    static void init() {
+        ms_layout.begin()
+            .add(bgfx::Attrib::Position, 3, bgfx::AttribType::Float)
+            .add(bgfx::Attrib::Color0, 4, bgfx::AttribType::Uint8, true)
+            .add(bgfx::Attrib::TexCoord0, 2, bgfx::AttribType::Float)
+            .end();
+    }
+
+    static bgfx::VertexLayout ms_layout;
```

Comment:
`inline static bgfx::VertexLayout ms_layout;` and then you don't have to re-declare it inside the source. Or outside the struct definition in this case.

Reply:
skybox vertex class is obsolete and is handled by instanceMesh